src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -1,10 +1,12 @@
-@echo off 
+@if "%_echo%" neq "on" echo off
 setlocal
 
-REM 
+REM
 echo Start WCF Self Hosted Service on local machine
 REM
 
+set WCFServiceCleanup=true
```

Comment:
If self-host server already started, WCFCustomSetup(who calls this script) will ping and exit.